steel/README.md

```diff
@@ -1,150 +1,56 @@
-        function balanceOf(address account) external view returns (uint);
-    }
-}
+# Steel - Boundless runtime for EVM apps
 
-/// Function to call, implements the `SolCall` trait.
-const CALL: IERC20::balanceOfCall = IERC20::balanceOfCall {
-    account: address!("9737100D2F42a196DE56ED0d1f6fF598a250E7E4"),
-};
+## Introducing Steel, a production-ready EVM execution prover
 
-/// Address of the deployed contract to call the function on (USDT contract on Sepolia).
-const CONTRACT: Address = address!("aA8E23Fb1079EA71e0a56F48a2aA51851D8433D0");
-/// Address of the caller. If not provided, the caller will be the [CONTRACT].
-const CALLER: Address = address!("f08A50178dfcDe18524640EA6618a1f965821715");
+Steel is a production-ready EVM execution prover designed to bring boundless runtime to all EVM apps. Using execution proofs, Steel enables EVM apps to run completely offchain, while preserving onchain security.  With Steel, you can prove correct smart contract execution without re-execution, allowing blockchain developers unbounded computation over on-chain data.. 
 
-fn main() {
-    // Read the input from the guest environment.
-    let input: EthEvmInput = env::read();
+Our partners are already developing game-changing applications with Steel.  One application has shown gas savings of 1.2 billion gas for a contract call using around 400,000 SLOADs. 1.2 billion gas is around 30 blocks worth of execution and this can be verified onchain in one proof, that costs under $10 to generate, and less than 300k gas to verify. Steel unlocks boundless application runtime, without rollups, without centralization, without re-writing your smart contracts, and without writing ZK circuits. The brakes are off.
 
-    // Converts the input into a `EvmEnv` for execution. The `with_chain_spec` method is used
-    // to specify the chain configuration. It checks that the state matches the state root in the
-    // header provided in the input.
-    let env = input.into_env().with_chain_spec(&ETH_SEPOLIA_CHAIN_SPEC);
-    // Commit the block hash and number used when deriving `EvmEnv` to the journal.
-    env::commit_slice(&env.commitment().abi_encode());
+## Getting Started with Steel
 
-    // Execute the view call; it returns the result in the type generated by the `sol!` macro.
-    let contract = Contract::new(CONTRACT, &env);
-    let returns = contract.call_builder(&CALL).from(CALLER).call();
-    println!("View call result: {}", returns._0);
-}
-```
+The recommended place is to start is [Steel examples], specifically the [ERC20 Counter] example. 
 
-## Host code
+The [create-steel-app] script will allow you to set up the erc20-counter example locally in one command:
 
-Here is a snippet to the [relevant code](../examples/erc20/host/src/main.rs) on the host, it requires the same arguments as the guest:
+`sh -c "$(curl -fsSL https://raw.githubusercontent.com/risc0/risc0-ethereum/refs/heads/main/steel/docs/create-steel-app/create-steel-app)"`
```

Comment:
It's better to use file-io based execution rather than args. We can't use pipe `|` since we need `stdin` (I assume this is why we are using `-c` here) but there is another way.
````suggestion
```
sh <(curl -fsSL https://raw.githubusercontent.com/risc0/risc0-ethereum/refs/heads/main/steel/docs/create-steel-app/create-steel-app)
```
````